package.json

```diff
@@ -33,14 +33,18 @@
     "dist/assets/styles/configuration.css"
   ],
   "dependencies": {
+    "@lit/context": "^1.1.1",
+    "@lit/task": "^1.0.0",
     "@xmpp/jid": "^0.13.1",
     "async": "^3.2.2",
     "decache": "^4.6.0",
     "escape-html": "^1.0.3",
     "got": "^11.8.2",
     "http-proxy": "^1.18.1",
+    "lit": "^3.1.3",
```

Comment:
The current version of ConverseJS uses lit 2.4.0. Is your code compatible with this version? If so, can we change for it, so we have the same version everywere? (would be easier for devs, so we don't have to know both).

I know you told me there are a lot of performance improvments in lit 3.x. But i'm not sure it is necessary for our usecase.

If you think v3.x is better, i'm ok. I don't have strong opinion on that. Just want to ease the dev process.

Reply:
Note: after discussions, i'm ok with lit 3.x